fingerprint/infra/nec-bio-sdk/src/main/java/com/simprints/fingerprint/infra/necsdkimpl/acquisition/template/FingerprintTemplateProviderImpl.kt

```diff
@@ -105,4 +105,4 @@ internal class FingerprintTemplateProviderImpl @Inject constructor(
 
 }
 
-const val NEC_TEMPLATE_FORMAT = "NEC_1"
+const val NEC_TEMPLATE_FORMAT = "NEC_1_5"
```

Comment:
Does this affect any of the existing templates?

Reply:
We didn't yet use NEC in any prod project